### DISCUSSION TITLE
Welcome back

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#145] [General]
Hi Engageub,

I am very happy that your hub is comeback. Many people are appreciate your work. Hope you will contribute more in the future and I just propose that if you have Discord channel, it will be great.
Thank you

**Comment by engageub** [comment]
Hello,
Thank you for your feedback.

The account in question was initially flagged due to multiple accounts being accessed from the same IP address. After consulting with GitHub, the flagging issue was resolved, as it was a common problem experienced by other users as well. Although the account was accessible, it was not visible to the public. If you would like to retain the code, you are welcome to fork the repository; this will ensure that you have access to the code even if the account is flagged again.

Regarding Discord, I understand the need for a dedicated space for discussions. Unfortunately, due to time constraints, I am unable to maintain and respond to all queries actively, as this is not my full-time endeavor. To address common questions and issues, I have created an FAQ section. 

The design of the project is complete, and the primary task now is to add new Docker images following the existing pattern. If you have suggestions for new applications, please let me know so they can be incorporated into the script. 

Thank you for your understanding and cooperation.

Thank you

**Reply by superit63** [reply]
Thanks a lot for your work. I am appreciate it
